client/app/models/disposition.js

```diff
@@ -186,4 +186,25 @@ export default class DispositionModel extends Model {
     // e.g. `conditional favorable`
     return this.get(PARTICIPANT_TYPE_RECOMMENDATION_TYPE_LOOKUP[participantType]);
   }
+
+  @computed('statecode', 'statuscode', 'dcpRepresenting', 'dcpIspublichearingrequired')
+  get showHearingDetails() {
+    if (
+      (['Active', 'Inactive'].includes(this.get('statecode')))
+          && (['Saved', 'Submitted', 'Not Submitted'].includes(this.get('statuscode')))
+          && (this.get('dcpIspublichearingrequired') === 717170000)
```

Comment:
Here you can import the `dcpIspublichearingrequired`  constant into the template, instead of using the raw value. 

something like 
` === this.dcpIsPublichearingrequiredConstant.YES`

see `client/app/components/archive-project-milestone-list-item.js`

Reply:
I created a constant file for dispositions here:
https://github.com/NYCPlanning/labs-zap-search/blob/develop/client/app/models/disposition/constants.js